_config.yml

```diff
@@ -2,15 +2,15 @@
 markdown: kramdown
 highlighter: rouge
 paginate: 100 # it will paginate after this number
-language: 'en-us'
+language: "en-us"
 words_per_minute: 150
 
 # edit baseurl to simply '/' if using as your personal page (instead of a project page)
-baseurl: '/'
+baseurl: "/"
 
 # Website info
 title: squ1rrel
-description: Vanderbilt's CTF Club
+description: The Vanderbilt University CTF Club
```

Comment:
```suggestion
description: Vanderbilt University CTF Club
```

Reply:
no I think it makes more sense with the the, bc squ1rrel is the vanderbilt university ctf club

Reply:
Yeah but it's cumbersome to have the definite article it's like we don't have "The Vandyhacks" but "Vandyhacks."